products/zoningtaxlots/models/intermediate/int__zoningtaxlots.sql

```diff
@@ -39,33 +39,34 @@ zonechange AS (
 insertion AS (
     SELECT
         a.id AS dtm_id,
-        (CASE
```

Comment:
Ah - you do need to rebuild your dev container to sqlfluff properly, sorry. This isn't using our config, and we differ from standard sqlfluff a bit. In this case. This is an int table so it's not necesssarily the worst thing in the world, but I don't want sqlfluff setting the query order

Reply:
The rule here that sqlfluff applies that we ignore is that calculated columns are selected after non-calculated columns. This is an insane rule to me

Reply:
So I think the easiest thing to do would be
- rebuild dev container (I believe this can be done solely by clicking the button in the bottom left of vscode to bring up dev container options, then selecting "rebuild container"
- drop the sqlfluff commit (easiest to do with a rebase - I can show you on a call)
- re-run sqlfluff fix and add the new commit

Then we should be good to go

Reply:
@fvankrieken I reverted the previous `sqlfluff fix` and re-ran it with the rebuilt dev container. It looks like some of the fixes I reverted, such as splitting long lines, aren't re-applied by the newly configured sqlfluff. Is that also ignored?

Reply:
Yup, we made the line length limit just a tad longer than default

Reply:
You can look through `pyproject.toml` for our sqlfluff custom config